pkg/virt-config/virt-config.go

```diff
@@ -46,6 +46,7 @@ const (
 	DefaultUseEmulation                             = false
 	DefaultUnsafeMigrationOverride                  = false
 	DefaultPermitSlirpInterface                     = false
+	DefaultConsoleServerPort                        = 8186
```

Comment:
Same as above, no need for an extra weberserver. We just need one for all our http traffic.